WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -1035,28 +1069,35 @@ import WordPressComAnalytics
                 DDLogSwift.logError("Error: Could not retrieve an existing topic via its objectID")
                 return
             }
+
             let objectID = topicInContext.objectID
-            service.fetchPostsForTopic(topicInContext,
-                earlierThan: NSDate(),
-                success: { [weak self] (count:Int, hasMore:Bool) in
-                    dispatch_async(dispatch_get_main_queue(), {
-                        if let strongSelf = self {
-                            if strongSelf.recentlyBlockedSitePostObjectIDs.count > 0 {
-                                strongSelf.recentlyBlockedSitePostObjectIDs.removeAllObjects()
-                                strongSelf.updateAndPerformFetchRequest()
-                            }
-                            strongSelf.updateLastSyncedForTopic(objectID)
-                        }
-                        success?(hasMore: hasMore)
-                    })
-                },
-                failure: { (error:NSError?) in
-                    dispatch_async(dispatch_get_main_queue(), {
-                        if let error = error {
-                            failure?(error: error)
+
+            let successBlock = { [weak self] (count:Int, hasMore:Bool) in
+                dispatch_async(dispatch_get_main_queue(), {
```

Comment:
Whenever the last parameter is a closure, you may pass it this way (no need to wrap it up inside the parenthesis):

```
dispatch_async(...) {
   // Closure!
}
```